src/reports/package/accessibilityInsightsReport.d.ts

```diff
@@ -1,19 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
+import axe from 'axe-core';
```

Comment:
Could this be `import * as axe from 'axe-core'` instead? I'm still seeing some build issues from the services side.

Reply:
let me see about that

Reply:
Changed as requested.